deploy/crds/csi.ibm.com_ibmblockcsis_crd.yaml

```diff
@@ -2,6 +2,13 @@ apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
   name: ibmblockcsis.csi.ibm.com
+  labels:
+    app.kubernetes.io/name: ibm-block-csi-operator
+    app.kubernetes.io/instance: ibm-block-csi-operator
+    app.kubernetes.io/managed-by: ibm-block-csi-operator
+    product: ibm-block-csi-driver
+    release: v1.3.0
```

Comment:
could you update to 1.4.0 where applicable?

Reply:
done

Reply:
are you sure? I think I still see [some](https://github.com/IBM/ibm-block-csi-operator/blob/dd4ddf86cc92864d2e9d8fb4bd230e6d695f58a2/deploy/olm-catalog/ibm-block-csi-operator/csi.ibm.com_ibmblockcsis_crd.yaml#L10)

Reply:
not same file

Reply:
I found the version 1.3.0 in file deploy/olm-catalog/ibm-block-csi-operator/csi.ibm.com_ibmblockcsis_crd.yaml
- updated